v21.1/use-cloud-storage-for-bulk-operations.md

```diff
@@ -4,28 +4,26 @@ summary: Import data into your CockroachDB cluster.
 toc: true
 ---
 
-CockroachDB uses the URL provided in a [`BACKUP`](backup.html), [`RESTORE`](restore.html), [`IMPORT`](import.html), or [`EXPORT`](export.html) statement to construct a secure API call to the service you specify. The URL structure depends on the type of file storage you are using.
+CockroachDB uses the URL provided in a [`BACKUP`](backup.html), [`RESTORE`](restore.html), [`IMPORT`](import.html), [`EXPORT`](export.html), or [`CREATE CHANGEFEED`](create-changefeed.html) statement to construct a secure API call to the service you specify. The URL structure depends on the type of file storage you are using. We strongly recommend using cloud/remote storage.
```

Comment:
This would be easier to parse in a listicle format e.g. something like

CockroachDB constructs a secure API call to the cloud storage service you specify in a URL passed to one of the following statements:
- FOO
- BAR
- etc

If we do strongly recommend using cloud/remote storage, I think we should make that sentence bold or in a note or in a bold note